linux/index.js

```diff
@@ -47,97 +51,67 @@ const start = () => {
     resizable: false,
     title: 'Headset',
     maximizable: false,
-    titleBarStyle: 'hiddenInset',
     useContentSize: true,
     icon: 'icon.png',
-    frame: true,
+    webPreferences: { nodeIntegration: true },
+  });
+
+  player = new BrowserWindow({
+    width: 427,
+    height: 300,
+    useContentSize: true,
+    show: false,
+    title: 'Headset - Player',
+    icon: 'icon.png',
+    webPreferences: { nodeIntegration: true },
   });
 
   mainWindowState.manage(win);
 
   if (isDev) {
     win.loadURL('http://127.0.0.1:3000');
+    player.loadURL('http://lvh.me:3001');
```

Comment:
We probably should use `lvh.me` everywhere. When loading from the local server, YouTube refuses to play some videos if it detects that's coming from  `127.0.0.1`

Reply:
The main window doesn't have this problem, so that's why only the embedded YouTube player uses `lvh.me`. I'm going to add a comment to this line so we know why, maybe that'll be enough?